docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
 ---
-title: Teleport Trusted Clusters
-description: How to configure access and trust between two Teleport Clusters.
+title: Configure Trusted Clusters
+description: Explains how you can configure a trust relationship and manage access between two Teleport clusters.
```

Comment:
I wonder if we need more guidance on how to formulate/phrase the description. We're pretty inconsistent right now. Many of them start with "How to". Some start with a verb ("learn the key components" or "manage access requests". Others are pretty generic "Comprehensive reference for XYZ".

Reply:
In most cases, I prefer starting with a verb. I'm not sure how important it is to standardize. SEO keywords would probably be more bang for the buck. But if all we have are descriptions, more descriptive statements would be better than the real terse ones.

I can add this to the style guide along with some other "best practices" I'd like to see us adopt.

Reply:
Since meta descriptions are for summarizing a page in search results (and to a lesser extent in social cards), I think we should adopt standards for that purpose. 

Maybe it's a matter of taste, but this seems incomplete to me:

<img width="637" alt="image" src="https://github.com/gravitational/teleport/assets/25250668/11b94af3-08df-4c05-915f-ab4dfb692240">